ktor-client/ktor-client-core/common/test/MultiPartFormDataContentTest.kt

```diff
@@ -0,0 +1,30 @@
+import io.ktor.client.request.forms.*
+import io.ktor.test.dispatcher.*
+import io.ktor.utils.io.*
+import io.ktor.utils.io.core.*
+import kotlin.test.*
+
+/*
+ * Copyright 2014-2020 JetBrains s.r.o and contributors. Use of this source code is governed by the Apache 2.0 license.
+ */
+
+class MultiPartFormDataContentTest {
+
+    @Test
+    fun testMultiPartFormDataContentHasCorrectPrefix() = testSuspend {
+        val formData = MultiPartFormDataContent(formData {
+            append("Hello", "World")
+        })
+
+        val channel = ByteChannel()
+        formData.writeTo(channel)
+        channel.close()
+
+        val actual = channel.readRemaining().readBytes()
+
+        assertNotEquals('\r', actual[0])
```

Comment:
Could be more readable if you compare a slice for the first 4 bytes instead of comparing individual bytes.

Reply:
Nope, cause it will be a comparison of byte arrays instead of chars